Fedoraware/TeamFortress2/TeamFortress2/Features/ESP/ESP.cpp

```diff
@@ -1248,6 +1249,7 @@ void CESP::DrawWorld() const
 
 					if (Vars::ESP::Main::Outlinedbar.Value)
 						g_Draw.OutlinedRect(x - 1, y - 1, w + 2, h + 2, Colors::OutlineESP);
+						g_Draw.OutlinedRect(x + 1, y + 1, w - 2, height - 2, Colors::OutlineESP);
```

Comment:
This expression isn't covered by the if-statement in line 1164

Reply:
it works in getterhook so it works here

Reply:
Try disabling Outlinedbar. You will see that it still runs the second OutlinedRect call even though it's not intended.